justfile

```
@@ -7,7 +7,7 @@ test pattern="": build
   cargo test --all -- --test-threads=1 {{ pattern }}
   rm -f 'filename with spaces' foo
 
-test_lib pattern="": build
+test_lib_fast pattern="":
```

You can use dashes in Just recipe names, which are slightly easier to type.

```suggestion
test-lib-fast pattern="":
```

---

Yeah, that's better.